core/src/test/java/greencity/controller/HabitControllerTest.java

```diff
@@ -30,9 +32,7 @@
 import org.springframework.test.web.servlet.MockMvc;
 import static greencity.ModelUtils.getPrincipal;
 import static org.mockito.Mockito.verify;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
```

Comment:
You should better add new import instead using asterisks in imports: 
import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;